babel.config.js

```diff
@@ -1,13 +1,9 @@
 module.exports = {
-  presets: [
-    '@babel/env',
-    '@babel/react',
-    '@emotion/babel-preset-css-prop',
-  ],
+  presets: ['@babel/env', '@babel/react', '@emotion/babel-preset-css-prop'],
```

Comment:
```suggestion
  presets: ['@babel/env', '@babel/react', '@emotion/babel-preset-css-prop', '@babel/preset-typescript'],
```

Reply:
we dont need typescript here, adding it makes the build fails, the plugin is set in the `src` folder which we run the command babel cli manually for the `lib` folder that contains the constants and utils. I think the issue adding in here is that it tries to convert 2 times since rollup also has the TS plugin